src/routers/comment.js

```diff
@@ -0,0 +1,17 @@
+const {Router} = require('express');
+const router = Router();
+const checkAuth = require('../middleware/auth');
+const checkRoleAuth = require('../middleware/role-auth');
+const {getComments, postComment, deleteComment} = require('../controllers/comment-controller');
+const {validationCreationComment, validationDeleteComment} = require('../validators/comment-validator');
+const {validateExistsAccount} = require('../controllers/account-controller');
+const {validateExistsDiscussion} = require('../controllers/discussion-controller');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
const {validateExistsDiscussion} = require("../controllers/discussion-controller");
```